### ISSUE TITLE
Add PhpCsFixer plugin results to Information/Errors tabs (Like PHPUnit and Codeception) 

### ISSUE THREAD

**Issue by corpsee** [php-censor/php-censor#74]
Add PhpCsFixer plugin results to Information/Errors tabs (Like PHPUnit and Codeception)

**Comment by SimonHeimberg** [comment]
Hint: With this command
`php-cs-fixers fix --diff -vvv --format json --dry-run ...`

you get such output:
``` json
{
   "files" : [
      {
         "name" : "xxx/Controller/DashboardController.php",
         "diff" : "--- Original\n+++ New\n@@ @@\n         $filterform->handleRequest($request);\n-        \n+\n         if ($fil...",
         "appliedFixers" : [
            "phpdoc_indent",
            "operators_spaces",
            "single_quote",
            "unalign_double_arrow",
            "pre_increment",
            "concat_without_spaces",
            "unalign_equals",
            "whitespacy_lines",
            "return",
            "extra_empty_lines",
            "braces"
         ]
      }
   ],
   "time" : {
      "total" : 0.654
   },
   "memory" : 8
}
```

Unfortunately the diff is not very nice (no line numbers, ...).

**Comment by corpsee** [comment]
Thanks. It is useful command.

**Comment by corpsee** [comment]
Implemented in #193.
